gles3_builders.py

```diff
@@ -499,7 +499,7 @@ def build_gles3_header(
             fd.write("\t\t};\n\n")
             variant_count = len(header_data.variant_defines)
         else:
-            fd.write("\t\tstatic const char **_variant_defines[]={" "};\n")
+            fd.write('\t\tstatic const char **_variant_defines[]={" "};\n')
```

Comment:
This was the offending line, as the original fix was:
```diff
-           fd.write("\t\tstatic const char **_variant_defines[]={" "};\n")
+           fd.write("\t\tstatic const char **_variant_defines[]={};\n")
```
The intent was to have the fallback container store a single value, but the outer double quotes meant Python was concatenating two strings instead, resulting in an empty container. Using single quotes restores the intended behavior.